docs/_data/sponsors.yml

```diff
@@ -0,0 +1,3 @@
+- name: Forestry.io
+  image: /img/forestry-logo.png
+  url: https://forestry.io
```

Comment:
personal nit: Can we have a blank newline here please?

Reply:
oh of course